.github/workflows/aa_basic.yml

```diff
@@ -69,19 +69,15 @@ jobs:
         run: |
           make && make install
 
-      - name: Musl build with default features
+      - name: Musl build with none platform
         run: |
-          make LIBC=musl
-
-      - name: s390x build with offline_fs_kbc feature
-        run:
-          make ARCH=s390x KBC=offline_fs_kbc
+          make LIBC=musl ATTESTER=none
 
       - name: Run cargo test
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --features openssl,rust-crypto,offline_fs_kbc,kbs,coco_as -p attestation-agent -p attester -p coco_keyprovider -p kbc -p kbs_protocol -p attestation_agent -p crypto -p resource_uri -p sev@0.1.0
+          args: --features openssl,rust-crypto,all-attesters,kbs,coco_as -p attestation-agent -p attester -p coco_keyprovider -p kbc -p kbs_protocol -p crypto -p resource_uri
```

Comment:
I don't see the `offline_fs_kbc` or `offline_sev_kbc` being tested in the CDH workflow. Do we need to add that?

Reply:
I will add some unit tests to `offline_fs_kbc`, which will be enabled by default for CDH without specifying features explicitly.

`offline_sev_kbc` will not be used so it will not be test.

The original code of `online_sev_kbc` does not have any tests. The [test](https://github.com/confidential-containers/guest-components/blob/main/.github/workflows/cdh_basic.yml#L67) will enable `sev` feature, which will cover the tests of  `online_sev_kbc`.